src/components/sections/challenges/Rewards.tsx

```diff
@@ -13,15 +13,17 @@ import RewardCertificate from "@/components/cards/challenge/RewardCertificate";
  * @export
  * @returns {ReactElement}
  */
-export function OverviewRewards(): ReactElement {
+export function OverviewRewards({ testId }: { testId?: string }): ReactElement {
   const { t } = useTranslation();
   const challenge = useSelector((state) => state.challenges.current);
   const router = useRouter();
   const token = challenge?.reward?.token || challenge?.rewards[0]?.token || "";
 
   return (
     <Section title={`${t("communities.overview.reward.title")}`}>
-      <p className="my-6 text-base text-secondary">{t("course.challenge.reward.certificate.description")}</p>
+      <p data-testid={testId} className="my-6 text-base text-secondary">
```

Comment:
why do you need this data-testId

Reply:
to test the content inside the reward as the section component is already tested

Reply:
can you use `getByText` to get the content instead of adding a `data-testid`